medusa/indexers/tmdb/tmdb.py

```diff
@@ -560,7 +560,7 @@ def get_last_updated_series(self, from_time, weeks=1, filter_show_list=None):
                     new_list.append(show.indexerid)
             total_updates = new_list
 
-        return total_updates
+        return list(total_updates)
```

Comment:
I'm not saying that there isn't valid reasons to return a list instead, for example iterating a list is faster than iterating a set, but checking contents or avoiding duplicates is faster than a list.  But unnecessarily converting types isn't an optimization.  I am just challenging to see if there is a valid reason.